targets/simple_switch/simple_switch.cpp

```diff
@@ -75,13 +76,23 @@ class SimpleSwitch::MirroringSessions {
   bool add_session(mirror_id_t mirror_id,
                    const MirroringSessionConfig &config) {
     Lock lock(mutex);
-    sessions_map[mirror_id] = config;
-    return true;
+    if (0 <= mirror_id && mirror_id <= RegisterAccess::MAX_MIRROR_SESSION_ID) {
```

Comment:
you must be the kind of person that uses `100[array]` for array access :)

Reply:
Only in an obfuscated code contest :)  But I'm missing what about that code change was odd enough to make you think that?

Reply:
I think it's just a mater of habit, I never write `0 <= x` instead of `x >= 0`, so my brain was frozen for a second.